sunpy/io/_fits.py

```diff
@@ -125,33 +125,50 @@ def get_header(afile):
     try:
         headers = []
         for hdu in hdulist:
-            try:
-                comment = "".join(hdu.header['COMMENT']).strip()
-            except KeyError:
-                comment = ""
-            try:
-                history = "".join(hdu.header['HISTORY']).strip()
-            except KeyError:
-                history = ""
-
-            header = FileHeader(hdu.header)
-            header['COMMENT'] = comment
-            header['HISTORY'] = history
-
-            # Strip out KEYCOMMENTS to a dict, the hard way
-            keydict = {}
-            for card in hdu.header.cards:
-                if card.comment != '':
-                    keydict.update({card.keyword: card.comment})
-            header['KEYCOMMENTS'] = keydict
-
-            headers.append(header)
+            headers.append(format_comments_and_history(hdu.header))
     finally:
         if close:
             hdulist.close()
     return headers
 
 
+def format_comments_and_history(input_header):
```

Comment:
This is a helper function for the test files or do we expect this to be useful elsewhere?

Reply:
I dunno, why?